config/rbac/role.yaml

```diff
@@ -72,6 +72,14 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - vpcresources.k8s.aws
```

Comment:
Where did we choose this group name?

Reply:
ack. Similar to karpenter.k8s.aws. Might be a little cleaner as vpc.k8s.aws, but that ship has sailed, I think.